openapi2jsonschema/command.py

```diff
@@ -153,7 +157,7 @@ def default(output, schema, prefix, stand_alone, expanded, kubernetes, strict):
 
             # These APIs are all deprecated
             if kubernetes:
-                if title.split(".")[3] == "pkg" and title.split(".")[2] == "kubernetes":
+                if title_split[3] == "pkg" and title_split[2] == "kubernetes":
```

Comment:
> These APIs are all deprecated

I just checked and this was dropped in k8s 1.14, so we can also drop this here :)